Automattic/jetpack
# Update/remove userless jargon

This PR replaces the `userless` we used to use to refer to the Site Connection.

It just replaces internal usage of the term and do not change any functionality

IMPORTANT: Do not merge this PR before we add support to the renamed `allow_site_connection` parameter on WPCOM. D61000-code

#### Changes proposed in this Pull Request:

* Removes the "user-less" jargon

#### Jetpack product discussion


p1HpG7-bRQ-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Are the tests passing?
* Spin up a site
* Test the connection in all states and the basic features. 
* Check the Jetpack Settings page and modules cards that are disabled on a site-level connection
* Look at the code and look for dangerous changes
